com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/CollectionBuilder.java

```diff
@@ -388,6 +389,7 @@ protected CollectionDesc.Builder transform(Task task, IResource resource, Collec
             // mergeSubCollections() embeds instances, but we want to count only "real" embeded instances
             if (embedIndex < countOfRealEmbededObjects) {
                 ComponentsCounter.countComponentsInEmbededObjects(project, genResource, compStorage);
+                goCount++;
```

Comment:
Count all the real embedded components wihtout collections (because each collection count its own GOs and components)